cmd/agent/app/start.go

```diff
@@ -279,6 +279,8 @@ func StopAgent() {
 	if common.Forwarder != nil {
 		common.Forwarder.Stop()
 	}
+	logs.Stop()
+	log.Info("logs-agent stopped")
```

Comment:
I'd move this log line into our own space, and start with a `stopping logs-agent`